### PR TITLE
docs: clarify what `otk.target` means

### PR DESCRIPTION
**Pull request by mvo5**
I found the existing explaination a bit hard to follow, this is my attempt to make it a bit clearer.

### CODE REVIEW

**Review by achilleas-k — APPROVED** [review]
I can't speak to the accuracy with respect to the original intent (I might have misunderstood it), but I do find the extra paragraph clarifies it the way I understand it.

**Review by supakeen — APPROVED** [review]
That's how it was meant. Thank you.
